torch/signal/windows/windows.py

```diff
@@ -98,6 +104,9 @@ def _window_function_checks(function_name: str, M: int, dtype: torch.dtype, layo
     {device}
     {requires_grad}
 
+Returns:
+    Tensor: A 1-D tensor of size :math:`(M,)` containing the window.
```

Comment:
This could probably go in the `window_common_args` dict. Also, it may render better as.
```suggestion
Returns:
    Tensor: A 1-D tensor of shape ``M`` containing the window.
```

Reply:
Yes, indeed. I'll add that.